src/App/Article/index.js

```diff
@@ -171,7 +171,9 @@ function Article({ id }) {
 
   const citations = uniq([
     ...(response.fields.cited_by || []),
-    ...(response.fields.citations_inbound || []).map(c => c.source_id),
+    ...(response.fields.citations_inbound || [])
+      .map(c => c.source_id)
+      .filter(c => !isNaN(c)),
```

Comment:
@jonmv Not sure if this is a bug, follow 2 citation_inbound entries for document 1557 do not have `source_id`
```
[
  {
    "neu": 1,
    "source_doi": "https://doi.org/10.1128/jvi.01774-19",
    "target_doi": "https://doi.org/10.1056/nejmoa1211721",
    "neg": 0.008267234,
    "target_id": 1557,
    "context": "The nearly complete genome sequence for MERS-CoV EMC (GenBank accession number JX869059) and PREDICT/PDF-2180 (GenBank accession number KX574227) were previously deposited in GenBank <cite data-doi=\"10.1128/mbio.00373-17\">(16,</cite><cite data-doi=\"10.1056/nejmoa1211721\">59)</cite>. The nearly complete genome sequence for MERS-CoV EMC (GenBank accession number JX869059) and PREDICT/PDF-2180 (GenBank accession number KX574227) were previously deposited in GenBank <cite data-doi=\"10.1128/mbio.00373-17\">(16,</cite><cite data-doi=\"10.1056/nejmoa1211721\">59)</cite>.",
    "pos": 0.013990303,
    "type": "mentioning"
  },
  {
    "neu": 1,
    "source_doi": "https://doi.org/10.1136/bjophthalmol-2020-315994",
    "target_doi": "https://doi.org/10.1056/nejmoa1211721",
    "neg": 0.012487674,
    "target_id": 1557,
    "context": "(MERS-CoV) was isolated in 2012 <cite data-doi=\"10.1056/nejmoa1211721\">6</cite> and has seen over 2400 cases reported to WHO to date, and over 850 deaths. <cite data-doi=\"10.3201/eid2602.190697\">7</cite> Of the 2223 laboratory-confirmed MERS-CoV cases reported to WHO, 415 were healthcare workers, representing over one-third of all secondary transmission.",
    "pos": 0.014867621,
    "type": "mentioning"
  }
]
```

Reply:
Not a bug — some of the citations from the scite data-set (the citations with source and target) had no articles with matching DOI in the data set.